src/com/tutorial/main/BasicEnemy.java

```diff
@@ -1,40 +0,0 @@
-package com.tutorial.main;
-
-import java.awt.Color;
-import java.awt.Graphics;
-import java.awt.Rectangle;
-
-public class BasicEnemy extends GameObject{
```

Comment:
So if I come back and decide to extend this somehow, I would just remove final, right?